CMakeLists.txt

```diff
@@ -111,7 +111,7 @@ endif()
 ######################################
 # BOOST
 ######################################
-find_package(Boost 1.74 COMPONENTS iostreams program_options REQUIRED)
```

Comment:
I'm not sure if this change is actually required. However 1.80 of beast did have a bug that made the application crash

Reply:
Easier said than done. There are no prebuilt binaries which means with this change along the build will take 10+ minutes longer when building all of boost. Of course this can be slightly optimized by only compiling the libraries that qlever needs but no compilation will always be faster than any compilation